lib/bowlingScoreCard.js

```diff
@@ -0,0 +1,62 @@
+class BowlingScoreCard {
+  constructor() {
+    this.eachRollScoreArray = [];
+  }
+
+  roll = (pinDown) => {
+    this.eachRollScoreArray.push(pinDown);
+  };
+
+  overallScore = () => {
+    let score = 0;
+    let rollNumber = 0;
+    for (let i = 0; i < 10; i++) {
+      if (this.isASpareFrame(rollNumber)) {
```

Comment:
Despite the complex use of indices in this method, the clear method names and simple structure allow for code to be easily followed and understood